NewareNDA/NewareNDA.py

```diff
@@ -171,6 +175,11 @@ def _read_nda_130(mm):
             # Check for an auxiliary record
             elif bytes[0:5] == b'\x00\x00\x00\x00\x65':
                 aux.append(_aux_bytes_to_list(bytes[4:]))
+
+    # Get active mass
+    [active_mass] = struct.unpack('<f', mm[-44:-40])

```

Comment:
I think the only reference point is this string.
`b'\x06\x00\xf0\x1d\x81\x00\x03\x00\x61\x90\x71\x90\x02\x7f\xff\x00'`
Maybe only the first 8 bytes are needed, but it has been reproducible on my end with all 16.